### DISCUSSION TITLE
Show model icon for every chat in the history list

### DISCUSSION THREAD

**Discussion by antonme** [danny-avila/LibreChat#64] [Feature Requests & Suggestions]
Currently, the only way to know which model was used for a chat in history is by clicking on the chat and then clicking on the model icon near the prompt. It would be much more convenient if the model icon was displayed directly in the history list itself. This feature wasn't very important when there was only one model/assistant available in the original ChatGPT interface, but here it's much more necessary.

**Comment by danny-avila** [comment]
yes, it would be more beneficial than having a chat message icon. Thank you for all the great ideas

**Reply by antonme** [reply]
Maybe it would be even better to use black and white mdi icons instead of prompt-line colorful ones
